src/molecules/TrysilPlug.js

```diff
@@ -64,7 +65,14 @@ TrysilPlug.propTypes = {
 	offset: PropTypes.number,
 	sources: PropTypes.arrayOf(PropTypes.object),
 	ratio: PropTypes.number.isRequired,
-	ALLCAPS: PropTypes.bool,
+	Heading: PropTypes.element,
```

Comment:
If we are sending in and header. Should we then send inn props or just add the props to the heading before sending inn? I see we have a default value here. But should we? Or should we have a component that has heading as mandatory? And then two components that uses trysilplug but with different headings? Im not sure. Just wanted to throw the idea out here

Reply:
If I trow in the default, it's backwards compatible.

Reply:
I'm open to adding the props before we send the component in.

Reply:
Removed the prop from propTypes and defaultProps

Reply:
If we keep the default then we need to support sending header props in anyways. So then its no use. Lets keep it as it is.